forest/src/cli/mod.rs

```diff
@@ -24,16 +24,29 @@ pub(super) fn cli() -> Result<Config, io::Error> {
                 .takes_value(true)
                 .help("A toml file containing relevant configurations."),
         )
+        .arg(
+            Arg::with_name("genesis")
+                .long("genesis")
+                .takes_value(true)
+                .help("The genesis CAR file"),
+        )
         .get_matches();
 
+    let mut cfg: Config = Config::default();
+
     if let Some(config_file) = app.value_of("config") {
         // Read from config file
         let toml = read_file_to_string(config_file)?;
 
         // Parse and return the configuration file
-        return Ok(read_toml(&toml)?);
+        cfg = read_toml(&toml)?;
     };
+
+    if let Some(genesis_file) = app.value_of("genesis") {
+        cfg.genesis_file = Some(genesis_file.to_owned());
```

Comment:
if you change others, you won't need to have this as a String I don't think, the reference should be enough (this doesn't actually matter though)